### PR TITLE
US618048: Updating copyright

### PR DESCRIPTION
**Pull request by tbroadbent22**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=618048

### CODE REVIEW

**Comment by michael-bryson** [comment]
@tbroadbent22 You seem to be updating the copyright headers manually, have you not used the license header plugin?
You can run `mvn license:check` to check license headers and `mvn license:format` to update them after you make pom changes.

By the way it is failing because of these 2 properties [here](https://github.com/CAFAudit/audit-service/blob/develop/caf-audit-service-client/pom.xml#L34), these are overwriting the properties from the main pom.

**Comment by tbroadbent22** [comment]
> @tbroadbent22 You seem to be updating the copyright headers manually, have you not used the license header plugin? You can run `mvn license:check` to check license headers and `mvn license:format` to update them after you make pom changes.
> 
> By the way it is failing because of these 2 properties [here](https://github.com/CAFAudit/audit-service/blob/develop/caf-audit-service-client/pom.xml#L34), these are overwriting the properties from the main pom.

Changing and/or removing properties and running mvn tasks made no difference, its complaining about a test using a generated file - testTransformSuccess(com.hpe.caf.auditing.plugins.XMLToJavaTransformTest)

**Comment by michael-bryson** [comment]
> > @tbroadbent22 You seem to be updating the copyright headers manually, have you not used the license header plugin? You can run `mvn license:check` to check license headers and `mvn license:format` to update them after you make pom changes.
> > By the way it is failing because of these 2 properties [here](https://github.com/CAFAudit/audit-service/blob/develop/caf-audit-service-client/pom.xml#L34), these are overwriting the properties from the main pom.
> 
> Changing and/or removing properties and running mvn tasks made no difference, its complaining about a test using a generated file - testTransformSuccess(com.hpe.caf.auditing.plugins.XMLToJavaTransformTest)

The failure I was talking about was the license header check failure. As you say you now have a test failure. The test was failing because you had 2 copyright headers in the AuditLog.java file so the test expected 2 headers in the transform result.

File: caf-audit-service-client/pom.xml [hunk 1]
```diff
@@ -31,11 +31,6 @@
     
     <inceptionYear>2015</inceptionYear>
```
Comment (michael-bryson) [inline caf-audit-service-client/pom.xml]: Also unnecessary.

**Review by michael-bryson — APPROVED** [review]

**Pull request merged**